plugins/destination/gremlin/client/spec.go

```diff
@@ -1,35 +1,50 @@
 package client
 
 import (
+	_ "embed"
 	"fmt"
 	"runtime"
 	"strings"
+
+	"github.com/invopop/jsonschema"
+	orderedmap "github.com/wk8/go-ordered-map/v2"
 )
 
 type Spec struct {
-	Endpoint string `json:"endpoint"`
-	Insecure bool   `json:"insecure"`
+	// Endpoint for the database. Supported schemes are `wss://` and `ws://`, the default port is `8182`.
+	Endpoint string `json:"endpoint" jsonschema:"required,minLength=1"`
```

Comment:
```suggestion
	Endpoint string `json:"endpoint" jsonschema:"required,pattern=^ws[s]://[^\n]+$"`
```

Reply:
@disq please also verify with tests that bad string format won't work while proper ones do work